en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc

```diff
@@ -212,6 +212,75 @@ To delete a user:
+
+. Click *Save*.
+. Log out of Composer, navigate to the sign-on URL you entered in the *New Identity Provider* page, and then log in through your identity provider to test the configuration.
+
+To enable SAML 2.0 IDP:
+
+Note that the file-based configuration of a SAML 2.0 is not supported.
+
+. In the MuleSoft Composer sidebar, click *Settings* and then click *Multiple IDP*.
+. On the *Identity Provider* page, click *Add Identity Provider* > *SAML 2.0*.
+. In the *New Identity Provider* page, complete the required fields:
++
+* *Name*: Enter a nickname for this IDP.
+* *Sign On URL*: Redirect URL provided by the IDP for sign in. For example, `https://example.com/sso/saml`.
+* *Sign Off URL*: URL to redirect signout requests, so users both sign out of Composer and have their SAML user’s status set to `signed out`.
+* *Issuer*: ID of the identity provider instance that sends SAML assertions.
+* *Public Key*: Public key provided by the identity provider, which is used to sign the SAML assertion. It is the `X509Certificate` value in the SAML Response.
+* *Audience*: An arbitrary string value that identifies your Composer organization. The typical value for this string is `<organizationDomain>.composer.mulesoft.com`.
+* *Single Sign On Initiation*: Specify whether SSO can be initiated by Composer, your identity provider (for example, Okta), or both.
+
+** The *Service Provider Only* option allows only the Composer to initiate SSO.
+** The *Identity Provider Only* option allows only your external identity provider to initiate SSO.
+** The *Both* option allows either Composer or your external identity provider to initiate SSO. +
+The default value for this setting for newly configured identity provider configurations is Both.
+
+. Optionally, expand *Advanced Settings*, and provide the following values:
++
+* *Username Attribute*: Field name in the SAML `AttributeStatements` that maps to username. If no value is configured, the `NameID` attribute of the SAML `Subject` is used (Note: this is outside the SAML `AttributeStatements`).
+* *First Name Attribute*: Field name in the SAML `AttributeStatements` that maps to `First Name`.
+* *Last Name Attribute*: Field name in the SAML `AttributeStatements` that maps to `Last Name`.
+* *Email Attribute*: Field name in the SAML `AttributeStatements` that maps to `Email`.
+* *Group Attribute*: Field name in the SAML `AttributeStatements` that maps to `Group`.
+* *Require encrypted SAMl assertions* checkbox: If enabled, the SAML assertions sent from the IdP must be encrypted and follow the guidelines mentioned in the Prerequisites.
+
+. Click *Save*.
+. Log out of Composer, navigate to the sign-on URL you entered in the *New Identity Provider* page, and then log in through your identity provider to test the configuration.
```

Comment:
Confirm that you can just say Composer, or does it need to be MuleSoft Composer?